dwave/system/coupling_groups.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import dwave_networkx as dnx
+
+
+def coupling_groups(hardware_graph):
+    """Generate groups of couplers for which :class:`.DWaveSampler` applies a limit on total
```

Comment:
Why tie this in to a particular sampler? It is a property of the QPU.

Reply:
Fixed.